pkg/expapi/types.go

```diff
@@ -74,8 +75,8 @@ type SubresourceReference struct {
 	Subresource string `json:"subresource,omitempty"`
 }
 
-// TargetConsumption is an object for specifying target average resource consumption of a particular resource.
-type TargetConsumption struct {
+// Consumption is an object for specifying average resource consumption of a particular resource.
+type Consumption struct {
```

Comment:
Maybe ResourceConsumption will be a better name? Consumption seems to be to broad for me.